app/controllers/registrations_controller.rb

```diff
@@ -13,4 +14,11 @@ def configure_permitted_parameters
   def after_inactive_sign_up_path_for(_resource)
     confirmation_required_path
   end
+
+  private
+  def verify_captcha!
+    verification = HCaptcha.verify(params["h-captcha-response"])
+
+    redirect_to new_user_registration_path, alert: "Captcha verification failed. Please try again." if verification.failed?
```

Comment:
The alert message is currently hardcoded. I don't know how much of the backend contains hard-coded strings like this, but it might be worth remembering that we need to change this message when we'll do i8n.

Reply:
Yeah - this is a good point. Can you put this in the our i18n pls @kntsoriano? Maybe create a file called `errors` and put it in there.